map/bookmark_helpers.cpp

```diff
@@ -69,7 +69,7 @@ std::map<std::string, BookmarkMatchInfo> const kFeatureTypeToBookmarkMatchInfo =
   {"amenity-cinema", {kml::BookmarkIcon::Entertainment, BookmarkBaseType::Entertainment}},
   {"amenity-nightclub", {kml::BookmarkIcon::Entertainment, BookmarkBaseType::Entertainment}},
   {"amenity-theatre", {kml::BookmarkIcon::Entertainment, BookmarkBaseType::Entertainment}},
-  {"leisure-playground", {kml::BookmarkIcon::Entertainment, BookmarkBaseType::Entertainment}},
+  {"leisure-playground", {kml::BookmarkIcon::Park, BookmarkBaseType::Park}},
```

Comment:
A park icon with a tree is not appropriate for a kids playground. As there is no good icon for it yet, let's remove this line from the code completely. And create a new issue to add missing bookmark icons for other types, including the removed one.

Reply:
I have removed the line completely and have also opened a new issue(#6950) with current issue referenced.